images/env-archlinux/Dockerfile

```diff
@@ -21,7 +21,8 @@ RUN pacman --noconfirm -S \
     libtool \
     m4 \
     make \
-    pkgconf
+    pkgconf \
+    util-linux
```

Comment:
Not sure, but maybe for arch it's here? [util-linux-libs](https://archlinux.org/packages/core/x86_64/util-linux-libs/)